pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+prefer-workspace-packages: true
```

Comment:
This setting has to go into `.npmrc`; this file can only contain the package list.